### ISSUE TITLE
Add DeepSeek Coder models

### ISSUE THREAD

**Issue by klink** [smallcloudai/refact#216]
https://deepseekcoder.github.io/

**Comment by klink** [comment]
what models exactly will be added? please provide official names and links @JegernOUTT

**Comment by klink** [comment]
what will be the capabilities of the model? is it only for code completion, or for fine-tuning and chat as well? 
what are the gpu requirements for self-hosting?

**Comment by JegernOUTT** [comment]
@klink 
https://huggingface.co/deepseek-ai/deepseek-coder-1.3b-base
code-completion / finetuning
10Gb+ for finetune, 7Gb+ for inference 

https://huggingface.co/deepseek-ai/deepseek-coder-5.7bmqa-base
code-completion / finetuning
20Gb+ for finetune, 15gb+ for inference 

https://huggingface.co/deepseek-ai/deepseek-coder-6.7b-base
code-completion / finetuning
20Gb+ for finetune, 15gb+ for inference

**Comment by JegernOUTT** [comment]
@klink FYI we've disabled `deepseek-ai/deepseek-coder-6.7b-base` for now 'cause we're facing some problems with the loss calculation
